metaci/release/models.py

```diff
@@ -17,6 +17,28 @@ def __str__(self):
         return self.name
 
 
+class ImplementationStep(models.Model):
+    description = models.ForeignKey(
+        "plan.Plan", on_delete=models.CASCADE, related_name="name"
+    )
```

Comment:
We should probably leave this out and just use step.plan.name when we need a name/description. It doesn't make sense to have a field named description which is actually storing a lookup to a Plan model